### ISSUE TITLE
[Single Modules] Evaluate Server Side Rendering

### ISSUE THREAD

**Issue by labkode** [Semantic-Org/Semantic-UI#2678]
I am receiving this error when using semantic-ui-transition through browserify

Uncaught TypeError: $module.transition is not a function

My code is this:

````
var $ = require('jquery');
$.fn.dropdown = require("semantic-ui-dropdown");
$.fn.transition = require("semantic-ui-transition");
var Mn = require("backbone.Marionette");

module.exports = Mn.ItemView.extend({
		tagName: "div",
		attributes: {
			id: "header-app",
		},
		ui: {
			menu: ".ui .dropdown",
		},
		template: "#header-app-template",
		onShow:function() {
			this.ui.menu.dropdown();
		}
});
```

**Comment by jlukic** [comment]
This is not enough code for me to reproduce the issue.

**Comment by labkode** [comment]
What do you need more ?

Imagine you have this HTML:

```
<div class="ui inline dropdown">
      <div class="text">today</div>
      <i class="dropdown icon"></i>
      <div class="menu">
        <div class="header">Adjust time span</div>
        <div class="active item" data-text="today">Today</div>
        <div class="item" data-text="this week">This Week</div>
        <div class="item" data-text="this month">This Month</div>
      </div>
</div>
```

And your JS file is:

```
var $ = require('jquery');
$.fn.dropdown = require("semantic-ui-dropdown");
$.fn.transition = require("semantic-ui-transition");

$("document").ready(function() {
    $(".menu").dropdown();
});
```

**Comment by jlukic** [comment]
Just read the readme please

**Comment by trigoesrodrigo** [comment]
I get that same error sometimes and I'm requiring the whole sui package, so maybe it is not related to single modules.

Btw, I'm also using Marionette and Browserify. 

I'll try to isolate the problem and provide a repo to test it.

**Comment by jlukic** [comment]
The `module.export` statements for single component repos are [automatically generated via gulp](https://github.com/Semantic-Org/Semantic-UI/blob/master/tasks/admin/components/create.js#L150). Since we're automating this across ~50 components.

**Comment by jlukic** [comment]
See #2816, this should be resolved in `2.0.8`
